src/components/output/custom-els/PinchZoom/index.ts

```diff
@@ -268,7 +268,7 @@ export default class PinchZoom extends HTMLElement {
   }
 
   /** Transform the view & fire a change event */
-  private _applyChange (opts: ApplyChangeOpts = {}) {
+  private _applyChange(opts: ApplyChangeOpts = {}) {
```

Comment:
I was using the standardjs style, but I dislike the space after the function name,

What lint rules are you using? Check them in.

Reply:
The rules are in our package.json. Surma's PR swaps them out for tslint.